propagation/b3/shared.go

```diff
@@ -10,13 +10,15 @@ var (
 	ErrInvalidSpanIDHeader       = errors.New("invalid B3 SpanID header found")
 	ErrInvalidParentSpanIDHeader = errors.New("invalid B3 ParentSpanID header found")
 	ErrInvalidScope              = errors.New("require either both TraceID and SpanID or none")
+	ErrInvalidScopeParent        = errors.New("ParentSpanID requires both TraceID and SpanID to be available")
 	ErrEmptyContext              = errors.New("empty request context")
 )
 
+// Default B3 Header keys
 const (
-	b3TraceID      = "X-B3-TraceId"
-	b3SpanID       = "X-B3-SpanId"
-	b3ParentSpanID = "X-B3-ParentSpanId"
-	b3Sampled      = "X-B3-Sampled"
-	b3Flags        = "X-B3-Flags"
+	TraceID      = "x-b3-traceid"
```

Comment:
good choice default to lc

Reply:
yes... this allows me to use same identifiers for gRPC and HTTP with lowest amount of overhead.